proto/provenance/marker/v1/tx.proto

```diff
@@ -192,6 +192,7 @@ message MsgMintRequest {
 
   cosmos.base.v1beta1.Coin amount        = 1 [(gogoproto.nullable) = false];
   string                   administrator = 2;
+  optional string          recipient     = 3;
```

Comment:
You don't need or want the `optional` here. Removing it and regenerating the code should make it name the field `Recipient` again (instead of the weird `XRecipient` thing it's got now). To check if it was provided, you'd just do `if len(msg.Recipient) > 0 {...}`.

It'd probably be good to put a comment on the field in order to communicate that it's optional, something like `// recipient is the optional address of the account to receive the newly minted funds. If omitted, the minted funds are placed in the marker's account. Can only be provided if the marker is active and the administrator has withdraw permissions.`